Ports/bzip3/package.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='bzip3'
+version='1.2.2'
+files="https://github.com/kspalaiologos/bzip3/releases/download/${version}/bzip3-${version}.tar.gz bzip3-${version}.tar.gz 19e8d379f48610f945a04a988fd0c330ff6613b3df96405d56bed35a7d216dee"
+auth_type='sha256'
+useconfigure='true'
+installopts=("PREFIX=${SERENITY_INSTALL_ROOT}/usr/local")
+configopts=(
+    "--disable-arch-native"
```

Comment:
Small nitpick: non-interpolated strings should use single quotes instead of double quotes

Reply:
I mainly want to avoid things like the following (but that may just be my dislike for nicely aligned things that are somehow inconsistent in other ways):

```
configopts=(
    '--disable-arch-native'
    "--prefix=${SERENITY_INSTALL_DIR}"
)
```

Anyways, feel free to ignore, we'll have to find some objective rules for this anyways if we ever get around to making the formatting automated and enforcing it.

Reply:
I think we semi-agreed on having this in single quotes for now, so this isn't actually resolved yet. Sorry for the confusion! 😅

Reply:
AFAIK we use single quotes in this case.

Reply:
lucky, lucky… 😔

Reply:
@konradekk Stop making these comments across PRs, you are sending out notifications to everyone.